Documentation/project-docs/standard-platform.md

```diff
@@ -23,12 +23,14 @@ The .NET Platform Standard version represents binary portability across platform
 
 
 ## Terms
+- **.NET Platform Standard** - versioned sets of the reference assemblies that all .NET Platforms must support.
```

Comment:
> all .NET Platforms must support

Later in the doc there's a whole section on "guard rails" which starts with "In order to support platforms that implement a subset of the reference assemblies in a .NET Platform Standard version"... so I'm confused ;)